<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dReadSetting</class>
 <widget class="QDialog" name="dReadSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(1,192,198)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;华文中宋&quot;;color:white</string>
   </property>
   <property name="text">
    <string>默认阅读器型号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="defaultlabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;华文中宋&quot;;color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;华文中宋&quot;;color:white</string>
   </property>
   <property name="text">
    <string>支持阅读器型号：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="defaultcheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;华文中宋&quot;;color:white</string>
   </property>
   <property name="text">
    <string>设置为默认阅读器</string>
   </property>
  </widget>
  <widget class="QComboBox" name="readercomboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="checktoolButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid white;background:rgb(1,192,198);color:white;font-size:15px;font-weight:bold</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QToolButton" name="canceltoolButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid white;background:rgb(1,192,198);color:white;font-size:15px;font-weight:bold</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>411</width>
     <height>146</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid white;background:rgb(1,192,198);color:white;font-size:15px;font-weight:bold</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>defaultlabel</zorder>
  <zorder>label_3</zorder>
  <zorder>defaultcheckBox</zorder>
  <zorder>readercomboBox</zorder>
  <zorder>checktoolButton</zorder>
  <zorder>canceltoolButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
